packages/component-library/src/CivicCard/cardMetaTypes.js

```diff
@@ -28,7 +28,10 @@ const cardMetaObjectProperties = {
     })
   ),
   // authors: PropTypes.arrayOf(PropTypes.string /* image url */)
```

Comment:
can delete this comment now

Reply:
Woops. Meant to. It's covered in the condition below